### PR TITLE
Set-up new repo pilz_common

### PR DESCRIPTION
**Pull request by martiniil**
The changes in code are only due to clang-format.

### CODE REVIEW

File: .github/workflows/industrial_ci_action.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  CATKIN_LINT: true
+  CATKIN_LINT_ARGS: '--strict'
+  CMAKE_ARGS: "-DCATKIN_ENABLE_CLANG_TIDY=true"
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        env:
+          - {ROS_DISTRO: melodic, ROS_REPO: testing}
```
Comment (SansoneG) [inline .github/workflows/industrial_ci_action.yml:16]: I suppose we will activate `noetic` in a future PR?

File: pilz_industrial_motion_testutils/package.xml [hunk 2]
```diff
@@ -6,14 +6,13 @@
...
   <maintainer email="c.henkel@pilz.de">Christian Henkel</maintainer>
-  <maintainer email="h.slusarek@pilz.de">Hagen Slusarek</maintainer>
   <maintainer email="i.martini@pilz.de">Immanuel Martini</maintainer>
 
   <license>Apache 2.0</license>
 
-  <url type="website">https://wiki.ros.org/pilz_industrial_motion</url>
-  <url type="bugtracker">https://github.com/PilzDE/pilz_industrial_motion/issues</url>
-  <url type="repository">https://github.com/PilzDE/pilz_industrial_motion</url>
+  <url type="website">https://wiki.ros.org/pilz_industrial_motion_testutils</url>
```
Comment (SansoneG) [inline pilz_industrial_motion_testutils/package.xml:13]: This wiki page does not exist yet

File: pilz_testutils/README.md [hunk 3]
```diff
@@ -3,4 +3,4 @@
 This package provides tools for performing googletests with asynchronous callbacks. For example, it can be used in tests
 including a ros::Subscriber or actionlib::SimpleActionClient with callbacks.
 
-See the [Code API](http://docs.ros.org/melodic/api/pilz_testutils/html/pilz_testutils/classtesting_1_1AsyncTest.html) for a technical description including a simple usage example.
+See the [Code API](http://docs.ros.org/noetic/api/pilz_testutils/html/pilz_testutils/classtesting_1_1AsyncTest.html) for a technical description including a simple usage example.
```
Comment (SansoneG) [inline pilz_testutils/README.md]: Link does not lead anywhere (yet)

File: README.md [hunk 4]
```diff
@@ -1 +1,2 @@
-# pilz_common
\ No newline at end of file
+# pilz_common
```
Comment (SansoneG) [inline README.md:2]: I suppose we should bring this a little more in line with other READMEs we have in our repos.
I will commit a suggestion soon.

File: .github/workflows/industrial_ci_action.yml (hunk 1 above)
Comment (martiniil) [reply .github/workflows/industrial_ci_action.yml:16]: Yes.

File: pilz_testutils/README.md (hunk 3 above)
Comment (martiniil) [reply pilz_testutils/README.md]: Yes, this is wrong. I will revert it.

**Comment by SansoneG** [comment]
GitHub didnt let me add a picture with the README right away, so I will add it in a coming commit

File: pilz_industrial_motion_testutils/package.xml (hunk 2 above)
Comment (martiniil) [reply pilz_industrial_motion_testutils/package.xml:13]: Thanks. I created it.

**Review by SansoneG — APPROVED** [review]
Looks good now!

**Pull request merged**